repl/src/main/scala-2.12/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -34,6 +34,8 @@ import scala.tools.nsc.interpreter.StdReplTags.tagOfIMain
 import scala.tools.nsc.util.stringFromStream
 import scala.util.Properties.{javaVersion, javaVmName, versionString}
 
+import org.apache.spark.repl.InactivityTimeout.{startInactivityTimer, stopInactivityTimer}
```

Comment:
Please import just InactivityTimeout and use InactivityTimeout.startInactivityTimer and InactivityTimeout.stopInactivityTimer where it is needed.